.github/workflows/dapp-up-check.yaml

```diff
@@ -0,0 +1,64 @@
+name: Dapp Up Check
+
+## Run this on workflow manual dispatch and cron
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch to check'
+        required: true
+        default: 'main'
+  # https://crontab.guru/#0_2_*_*_*
```

Comment:
Could we also make this run on PR against `main`? Making sure newly added dapp urls work.